script.js

```diff
@@ -2,6 +2,9 @@ const menu = document.querySelector('.menu');
 const ham = document.querySelector('.ham');
 const xIcon = document.querySelector('.xIcon');
 const menuIcon = document.querySelector('.menuIcon');
+const buttons = document.querySelectorAll('.button');
+const exitButtons = document.querySelectorAll('.x');
+const pupup = document.querySelector('#popup');
```

Comment:
**Kindly work on these requested changes:**
1. The JavaScript code implements an array of objects to store JavaScript objects containing the information for each project.
2. Each object contains at least the following information about the project: name, description, featuring image, technologies, link to live version, and link to the project source code. Ie:

```
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 plataformer game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```

1. - The projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for projects list should be removed and instead it should be added via JavaScript.
2. - Modal popup data is populated with the data from the projects array.
3. - Thanks to the previous 2 points, adding projects in the future will be easy. Students will only add it in one place - as another object in the projects array.
4. - The JavaScript code is implemented following the best practices.